### PR TITLE
Console: fill URL in CLI mode

### PR DESCRIPTION
**Pull request by f3l1x**
I hope this will fix it. ;-)

@TomasVotruba @filipsuk Test it first, please. :taco: :tada:

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Thanks for PR! :clap: 

Could you also prepare PHPUnit test to prevent this in the future?

I can handle it, if you're not in the mood.

**Comment by f3l1x** [comment]
Sure, I'll prepare it, maybe tomorrow.

**Comment by TomasVotruba** [comment]
Thank you!

File: app/config/extensions/console.neon [hunk 1]
```diff
@@ -1,5 +1,8 @@
 extensions:
-    - Contributte\Console\DI\ConsoleExtension
+    console: Contributte\Console\DI\ConsoleExtension
+
+console:
+    url: https://eventigo.cz
```
Comment (filipsuk) [inline app/config/extensions/console.neon]: @f3l1x Thanks for the PR 👍  

Could we use this parameter: https://github.com/eventigo/eventigo-web/blob/master/app/config/parameters.neon#L2 ?

(the idea is, someone could use this app on different domain)

**Comment by filipsuk** [comment]
Could this be prepared before Thursday please (before sending next newsletter)? Otherwise I need to rollback or do some hacks in production. Test is not a must (however it would be nice to have a first test :D).

**Comment by TomasVotruba** [comment]
@filipsuk Good to know :+1: 

@f3l1x Please, let me know, if you can make it today. I would take care of it otherwise.

**Comment by f3l1x** [comment]
@TomasVotruba Sorry, I can't make it today.

**Comment by TomasVotruba** [comment]
@f3l1x [No problem](https://www.youtube.com/watch?v=gbrbUfYSt0E)! Thank you for fast reply.

(pulling up sleeves...) :hammer:

**Pull request merged**